content/projects/gsoc/2020/projects/machine-learning.adoc

```diff
@@ -90,7 +90,8 @@ https://medium.com/analytics-vidhya/embed-jupyter-notebook-into-static-webpages-
   * Time - 03:00 am (UTC)
 
 === Other links
-
+https://github.com/jenkinsci/machine-learning-plugin.git[Github] +
```

Comment:
:rocket: thanks!!! Now this page contains every resource related to the GSoC ML plugin.